backend/oas/provider/schemas/jobs.yaml

```diff
@@ -0,0 +1,211 @@
+jobs.JobId:
+  type: string
+  format: string
+  example: "7af020f6-2e38-4d70-8cf0-4349650ea08c"
+
+jobs.JobStatus:
+  type: string
+  enum:
+    - submitted
+    - ready
+    - running
+    - success
+    - failed
+    - cancelled
+  example: "submitted"
+
+jobs.JobInfoSampling:
+  type: object
+  description: "The descriptor of sampling jobs"
+  properties:
+    job_type:
+      type: string
+      enum: ["sampling"]
```

Comment:
I want the form of enum to be the same as estimation 
- "sampling"

Reply:
I agree and finished to modify. Other enum fields are also modified.